rasa_nlu/data_router.py

```diff
@@ -276,6 +277,8 @@ def get_status(self):
         # be other trainings run in different processes we don't know about.
 
         return {
+            "max_training_processes": self._training_processes,
```

Comment:
What is the semantic behind this? E.g. what does it mean to set `max_training_processes ` to `5`?

I think there is either an issue in the way I think about this or the way it is implemented. I though the intention was to allow 5 parallel training processes **per project**. The way it is implemented allows 5 process **in total**.

Reply:
The pool is created here, setting it to 5 creates 5 process that will receive training requests in a round robin approach. https://github.com/RasaHQ/rasa_nlu/blob/master/rasa_nlu/data_router.py#L108

This is part of @PHLF's original changes. We're not changing that, just removing the constraint that restricted it to be 1 training process per project.